### PR TITLE
Add ts-reaktive-aws module

### PR DESCRIPTION
**Pull request by vizog**
Add AWS auto-scaling groups support for akka clustering

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -217,6 +229,10 @@ lazy val `ts-reaktive-kamon-akka-cluster` = project
   .settings(commonSettings: _*)
   .settings(kamonSettings: _*)
 
+lazy val `ts-reaktive-aws` = project
+  .settings(commonSettings: _*)
+  .settings(awsSettings: _*)
+
 lazy val root = (project in file(".")).settings(publish := { }, publishLocal := { }).aggregate(
```
Comment (jypma) [inline build.sbt]: you need to add `ts-reaktive-aws` to the aggregate so it gets published, tested, etc. along with the rest.

**Comment by jypma** [comment]
actually, you can't even `project ts-reaktive-aws` without it, it seems :)

**Comment by vizog** [comment]
It is odd because I ran the tests locally. I'll fix it right away

File: build.sbt (hunk 1 above)
Comment (vizog) [reply build.sbt]: @jypma done

**Pull request merged**